cmd/list/kubeletconfig/cmd_test.go

```diff
@@ -29,7 +29,7 @@ var _ = Describe("List KubeletConfig Command", func() {
 			Expect(cmd.Use).To(Equal(use))
 			Expect(cmd.Short).To(Equal(short))
 			Expect(cmd.Long).To(Equal(long))
-			Expect(cmd.Aliases).To(ContainElements(alias))
+			Expect(cmd.Aliases).To(ContainElements("kubeletconfig", "kubelet-config", "kubelet-configs"))
```

Comment:
can we use ContainElements(aliases) here?

Reply:
@marcolan018 Good point. I've made that change.